### PR TITLE
Update ACL and SQL for modules using CLI

### PR DESCRIPTION
**Pull request by yurige-matrix**
Hi @bradymiller .Also made a new ability to install or update modules using the command line. To do this, I transferred all the actions to new functions.
Example: 
`/openemr/interface/modules/zend_modules/public/index.php zfc-module <moduleAction> <moduleName>`:
> zfc-module - part of route for console call
> moduleAction - install_sql, install_acl, upgrade_acl, upgrade_sql
> moduleName - name of module, for example: PatientFilterEventHookTest

### CODE REVIEW

File: interface/modules/zend_modules/public/index.php [hunk 1]
```diff
@@ -35,6 +35,11 @@
     }
 }
 
+if (php_sapi_name() === 'cli') {
+    $ignoreAuth = true;
+    $_GET['site'] = 'default';
```
Comment (bradymiller) [inline interface/modules/zend_modules/public/index.php]: Would be much better to not hard-code the site variable. Recommend allowing to set this when call via command line (if it is not set at command line, then would set it to 'default'. This then means that folks that are using multisite can use it.

**Comment by sjpadgett** [comment]
Just so I don't have to start tracing and testing code would you give some bullet points for these recent module updates. Couple practical examples perhaps. I'm trying to determine how much this would apply with what i'm doing with oe-modules etc.

File: interface/modules/zend_modules/public/index.php (hunk 1 above)
Comment (yurige-matrix) [reply interface/modules/zend_modules/public/index.php]: Hi @bradymiller. Hard-code has been removed.

**Comment by yurige-matrix** [comment]
> Just so I don't have to start tracing and testing code would you give some bullet points for these recent module updates. Couple practical examples perhaps. I'm trying to determine how much this would apply with what i'm doing with oe-modules etc.

Hi @sjpadgett . 
New path to run install or upgrade SQL or ACL files in console.
`php /openemr/interface/modules/zend_modules/public/index.php zfc-module --site=default --modaction=install_sql --modname=PatientFilterEventHookTest`
When you run without parameters, the script will show help on the parameters.

**Comment by bradymiller** [comment]
hi @yurige-matrix , Code looks very good. Just let me know that it's testing well for you, and then I'll bring it into the codebase. thanks! -brady

**Comment by yurige-matrix** [comment]
Hi @bradymiller. Everything is good, you can it bring it into the codebase now.

**Comment by bradymiller** [comment]
hi @yurige-matrix . Great, bringing into codebase now. Thanks for the awesome improvement! -brady

**Pull request merged**